### PR TITLE
Added license to OpenFlexure project

### PR DESCRIPTION
**Pull request by samdrea**

### CODE REVIEW

File: DeviceAdapters/OpenFlexure/OpenFlexure.cpp [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
 // FILE:          OpenFlexure.cpp
 // PROJECT:       Micro-Manager
 // SUBSYSTEM:     DeviceAdapters
 //-----------------------------------------------------------------------------
-// DESCRIPTION:   Adapter for the OpenFlexure Microscope. This adapter is used on the v5 Sangaboard.
+// DESCRIPTION:   Adapter for the OpenFlexure Microscope. This adapter is used on the v5 Sangaboard. 
+//                
+// AUTHOR:        Samdrea Hsu, samdreahsu@gmail.com, 06/22/2024
+//                
+//                Karl Hoover (stuff such as programmable CCD size  & the various image processors)
```
Comment (henrypinkard) [inline DeviceAdapters/OpenFlexure/OpenFlexure.cpp:10]: @samdrea did you mean to include these other authors?

Comment (samdrea) [reply DeviceAdapters/OpenFlexure/OpenFlexure.cpp:10]: @henrypinkard No, I'm not sure how that got in there... Could you edit them off or should I open a new pr?

Comment (henrypinkard) [reply DeviceAdapters/OpenFlexure/OpenFlexure.cpp:10]: If you make new commits and push again they should be added to the PR

**Comment by henrypinkard** [comment]
Great!

**Pull request merged**